jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-template.yaml

```diff
@@ -22,6 +22,8 @@
             nodenumber=5
             networkingmode=vxlan
             tempestoptions={tempestoptions}
+            nics=2
+            crowbar_networkingmode=team
```

Comment:
in mkcloud:73 we could add `[[ $crowbar_networkingmode = team ]] && nicsdefault=2`
to make it easier to use